src/common/types/metadata.js

```diff
@@ -132,6 +132,7 @@ type MetadataTemplateInstance = {
     id: string,
     fields: MetadataTemplateInstanceField[],
     scope: string,
+    templateId?: string,
```

Comment:
Do we need to change type of `MetadataTemplateInstance` in `metadata-editor` as well? Or change in `metadata-editor` and import in `Metadata.js`? In other component we import types directly from `metadata-editor` but if I remember correctly this method `createTemplateInstance` was created before we could import

Reply:
You are right, it seem to be to much hustle with additional attribute. Although, I think MetadataTemplateInstance id field should reflect id of an instance, no a template, so I'll keep this change